docs/docs/Versioning_And_Upgrade/Lua_Toolkit_To_SDK_Plugin_Migration/Converting_Migration_Scripts.md

```diff
@@ -0,0 +1,56 @@
+# Converting Lua Upgrade Scripts to Python Data Migrations
+To convert migrations (a.k.a. "upgrade scripts") that were originally written in lua we need to get the version that the migration upgrades from, the object type the migration is written for, and lastly convert the code into python code using the [decorators](Decorators.md) described previously.
+
+## Example
+Assume there are two versions of a lua toolkit, 1.0.0 and 1.1.0 where the 1.1.0 version is following the basic toolkit directory structure (actually containing all operations):
```

Comment:
A little difficult to figure out what this example is trying to say. Should the python plugin provide the same migration as Lua 1.1.0 version or it should provide both 1.0 migration as well as 1.1 migration. May be include both decorators?

Reply:
updated to make it more clear that python migration still needs to be written to go from 1.1 lua toolkit to python plugin